<?xml version="1.0" ?>
<testsuite name="ToDo-20230604145639" tests="1" time="17.162" failures="0" errors="1">
	<testcase classname="ToDo" name="test_counter" time="17.162">
		<error type="AssertionError" message="&lt;selenium.webdriver.remote.webelement.Web[102 chars]_4&quot;)&gt; != 0"><![CDATA[Traceback (most recent call last):
  File "/home/magdalena/webapp-test/test_script.py", line 122, in test_counter
    self.assertEqual(counter, 0)
AssertionError: <selenium.webdriver.remote.webelement.Web[102 chars]_4")> != 0
]]></error>
	</testcase>
	<system-out><![CDATA[<selenium.webdriver.remote.webelement.WebElement (session="e1b8038690ae8c7b945f5c6caa9f0cb3", element="3035BF4C91A57F44E0F2B335E99EE483_element_4")>
<class 'selenium.webdriver.remote.webelement.WebElement'>
]]></system-out>
	<system-err><![CDATA[]]></system-err>
</testsuite>
